TradeItIosTicketSDK2/TradeItLauncher.swift

```diff
@@ -2,7 +2,7 @@
     let linkBrokerUIFlow = TradeItLinkBrokerUIFlow()
     let tradingUIFlow = TradeItTradingUIFlow()
     let viewControllerProvider = TradeItViewControllerProvider()
-    let deviceManager = TradeItDeviceManager()
+    var deviceManager = TradeItDeviceManager()
```

Comment:
@mitochondrion had to make this `var` so the test could replace it with a fake. Don't feel good about that though. I could create some provider but it seems messy too.